scout/server/blueprints/variant/templates/variant/cancer-variant.html

```diff
@@ -286,27 +286,8 @@ <h6 class="card-header">Variant callers</h6>
 {% block scripts %}
   {{ super() }}
   {{ external_scripts() }}
+  {{ variant_scripts() }}
   <script>
-    var triggerGenesTabList = [].slice.call(document.querySelectorAll('#nav-genes-tab'))
```

Comment:
I really like the idea to reuse this code in all types of variant! 💯  but looking at other blueprints we had the convention of putting JS code inside the static folder of the blueprint. Examples:

<img width="270" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/5bcb5077-4107-44c5-81e2-ec45f5126ae4">

<img width="243" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/122be22c-1979-462f-b1d8-220eb587a5b7">


Is it doable to do the same here? Otherwise it is fine to have it like this

Reply:
Well, it would sure be doable, but we also use macros for common js since way back when. The separate js file dl has the downside of linter js check warnings for unused functions, but perhaps advantage of better editor env with a "pure" js file. I dont mind much either way, but the code is moving from one html template to another, and the prior already had js containing macros that were imported to the latter.